rosdep/base.yaml

```diff
@@ -5638,10 +5638,9 @@ libtins-dev:
 libtool:
   arch: [libtool]
   debian:
-    buster: [libtool, libltdl-dev, libtool-bin]
+    '*': [libtool, libltdl-dev, libtool-bin]
     lenny: [libtool, libltdl3-dev]
     squeeze: [libtool, libltdl-dev]
-    stretch: [libtool, libltdl-dev, libtool-bin]
```

Comment:
Actually, I think that all of the specifically-listed Debian distributions here are End-of-Life.  Thus I'd suggest we just simplify this key to:

```
  debian: [libtool, libltdl-dev, libtool-bin]
```

While we are in here, we may as well update the `ubuntu` key the same way:

```
  ubuntu: [libtool, libtldl-dev, libtool-bin]
```

Reply:
Totally agree.
I thought to do this but did not know the policy for deleting old keys.

Reply:
> I thought to do this but did not know the policy for deleting old keys.

To be honest, we don't have a hard-and-fast rule about removing support for old distributions.  Every time we do, somebody complains.

But in this case, the old distros are *very* old, so I think removing them is fine.